.gitignore

```diff
@@ -85,6 +85,7 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots/**/*.png
 fastlane/test_output
+fastlane/README.md
```

Comment:
This mustn't have been done before and no one ever run the tests locally via Fastlane, which is something I did investigating a CI failure.

Sort of out of scope, but small enough to fit in here anyway.